Form/EventListener/FixCheckboxDataListener.php

```diff
@@ -36,16 +36,18 @@ public static function getSubscribedEvents()
     }
 
     /**
+     * NEXT_MAJOR: remove this method.
```

Comment:
We can't use `@deprecated` in general. What if we add a typecast to a method parameter? This is no deprecation. `NEXT_MAJOR` is the fallback for everything we can't deprecate.